rewrite-java/src/main/java/org/openrewrite/java/tree/J.java

```diff
@@ -1053,6 +1056,46 @@ public EnumValueSet getEnumValues() {
             return new FindType(clazz).visit(this);
         }
 
+        public static J.ClassDecl addAnnotation(
+                J.ClassDecl c,
```

Comment:
You can remove this parameter `c`, as it should be able to refer to `this`. Suggestion is to make this non-static.